v2/AdminCore.Services/EventService.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using AdminCore.Common.Interfaces;
+using AdminCore.Constants.Enums;
+using AdminCore.DAL.Models;
+using AdminCore.DTOs.Employee;
+using AdminCore.DTOs.Event;
+namespace AdminCore.Services
+{
+  public class EventService : IEventService
+  {
+    public List<EventDto> FindByType(EventTypes EventType)
```

Comment:
Can you reimplement the interface? just to keep the implementations using IList too.